src/components/textfieldControl/js/Textfield.js

```diff
@@ -0,0 +1,138 @@
+/*
+Copyright 2013-2016 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+var fluid_3_0_0 = fluid_3_0_0 || {};
+
+(function ($, fluid) {
+    "use strict";
+
+    /*************
+     * Textfield *
+     *************/
+
+    /**
+     * A component for controlling a textfield and handling data binding.
+     * Typically this will be used in conjunction with a UI control widget such as
+     * button steppers or slider.
+     */
+    fluid.defaults("fluid.textfield", {
+        gradeNames: ["fluid.viewComponent"],
+        ariaOptions: {
+            // Specified by implementor
+            // ID of an external label to refer to with aria-labelledby
+            // attribute
+            // "aria-labelledby": ""
```

Comment:
Shall `aria-controls` be documented too? Same comment applies to `fluid.textfieldSlider`.